{{cookiecutter.project_slug}}/docs/conf.py

```diff
@@ -170,13 +176,10 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, '{{ cookiecutter.project_slug }}',
-     u'{{ cookiecutter.project_name }} Documentation',
+    (master_doc, "{{ cookiecutter.project_slug }}",
+     u"{{ cookiecutter.project_name }} Documentation",
      author,
-     '{{ cookiecutter.project_slug }}',
-     'One line description of project.',
-     'Miscellaneous'),
+     "{{ cookiecutter.project_slug }}",
+     "One line description of project.",
+     "Miscellaneous"),
```

Comment:
Why did you change to double quotes? Stylistic or other reason

Reply:
Just stylistic reasoning